### DISCUSSION TITLE
open-meteo

### DISCUSSION THREAD

**Discussion by rsssxd** [breezy-weather/breezy-weather#149] [Q&A] [answered]
When an open-mateo API is selected for weather data in the app, will the app continue to pull data through Accuweather?

**Comment by papjul** [comment]
Your question is a bit too vague, define a step-by-step (there can be different meanings for "select weather data").

Each location you add to your location list is bound to one provider (in the future, you will be able to optionally mix providers), it’s written in color on each location.

**Comment by rsssxd** [comment]
I chose Open Meteo, but it looks like the app is using data from other weather providers.

**Reply by papjul** [reply, marked as answer]
Then, do the search again and make sure to select Open-Meteo on bottom of the location search page. Then delete the old location associated with AccuWeather.
